src/components/PublicRoute.tsx

```
@@ -0,0 +1,14 @@
+import React, { useContext } from 'react'
+import { Navigate } from 'react-router-dom'
```

ESLint

[error] 'Navigate' is defined but never used.

(@typescript-eslint/no-unused-vars)